src/NodaTime.Web/Markdown/unstable/text.md

```diff
@@ -71,6 +71,23 @@ specifiers being available for some types but not others.
 - [ZonedDateTime patterns](zoneddatetime-patterns)
 - [Period patterns](period-patterns)
 
+Standard and custom patterns
+---------------
+
+*Standard* patterns are those denoted with a single character to represent a common pattern within the culture
+being used. For example, the standard pattern `d` for a `LocalDate` is in month/day/year format in a US culture,
+but day/month/year format in a UK culture. They're usually a shorthand for a possibly-culture-specific custom format,
+but not always. (Some standard patterns in Noda Time can't be represented directly in custom patterns.)
+
+*Custom* patterns give more direct control over how a value is formatted or parsed. It may
+still be culture-sensitive like standard patterns, but in a lower level way - the `/` format specifier within a `LocalDate` pattern is used to indicate
+the culture-sensitive date separator, which is `/` in a US culture but `.` in German culture, for example.
+
+When a single character is specified for a pattern, it is *always* treated as a standard pattern. If no standard
```

Comment:
Could almost bold this first sentence (or perhaps "are those **denoted with a single character** to represent" above). At least, this was the part that tripped me up for a while.